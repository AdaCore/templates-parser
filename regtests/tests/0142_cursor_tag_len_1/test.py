from test_support import *

gprbuild('cursor_tag_l1')
run(['bin/main', '5'])
run(['bin/main', '1'])
