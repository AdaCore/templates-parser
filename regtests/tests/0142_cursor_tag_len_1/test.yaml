# Needed even if empty, checked by the testsuite driver
