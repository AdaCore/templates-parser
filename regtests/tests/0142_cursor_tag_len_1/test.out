N: 5
     1  2  3  4  5

     1
     2
     3
     4
     5

N: 1
     1

     1

